src/schedule/timer_domain.c

```diff
@@ -231,9 +231,20 @@ static void timer_domain_set(struct ll_schedule_domain *domain, uint64_t start)
 	uint64_t ticks_tout = domain->ticks_per_ms * timer_domain->timeout /
 			     1000;
 	uint64_t ticks_set;
+	uint64_t ticks_now = platform_timer_get(timer_domain->timer);
```

Comment:
Is obtaining ticks_now actually required to identify that we are late? I suppose we can just compare ticks_req and ticks_set to decide whether we are late.